content/en/docs/Reference/file-based-catalogs.md

```diff
@@ -304,6 +304,48 @@ properties:
     kind: Bar
 ```
 
+#### `olm.deprecations`
+
+Operator authors can use the file-based catalog (FBC) deprecation schema to provide relevant information for support and upgrades for users running operators. The deprecation schema follows the Meta FBC Schema conventions. 
+
+Deprecation for packages/bundles/channels is enabled through the olm.deprecations schema. This schema consists of a list of scoped references to catalog content, with a descriptive message string that provides more information to the user. The deprecation schema is an optional feature.
```

Comment:
Since it is fenced in with backticks in the title, we should probably do the same for consistency here:
```suggestion
Deprecation for packages/bundles/channels is enabled through the `olm.deprecations` schema. This schema consists of a list of scoped references to catalog content, with a descriptive message string that provides more information to the user. The deprecation schema is an optional feature.
```